osu.Game/Skinning/LegacyJudgementPieceNew.cs

```diff
@@ -104,9 +104,8 @@ public void PlayAnimation()
                     this.ScaleTo(1.6f);
                     this.ScaleTo(1, 100, Easing.In);
 
-                    //todo: this only applies to osu! ruleset apparently.
-                    this.MoveTo(new Vector2(0, -2));
-                    this.MoveToOffset(new Vector2(0, 20), fade_out_delay + fade_out_length, Easing.In);
+                    this.MoveTo(new Vector2(0, -5));
+                    this.MoveToOffset(new Vector2(0, 80), fade_out_delay + fade_out_length, Easing.In);
```

Comment:
Stable source says:

```csharp
                    if (SkinManager.UseNewLayout && Player.Mode == PlayModes.Osu)
                        p.Transformations.Add(new Transformation(TransformationType.Movement, h.EndPosition + new Vector2(0, -5), h.EndPosition + new Vector2(0, 40), AudioEngine.Time, AudioEngine.Time + PostEmpt + HitFadeOut, EasingTypes.In));
```

`PostEmpt = 500`, `HitFadeOut = 600`. So you seem to have gotten the initial position right, but I don't know what the thing with the final position is.

Reply:
Yeah, I'm not sure here either. Stable says 40, not 80. But it visually looks close enough so I'm not going to nitpick for now.